app/src/test/java/org/schabi/newpipe/player/playqueue/PlayQueueTest.java

```diff
@@ -0,0 +1,186 @@
+package org.schabi.newpipe.player.playqueue;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+import org.schabi.newpipe.extractor.stream.StreamInfoItem;
+import org.schabi.newpipe.extractor.stream.StreamType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
+@RunWith(Enclosed.class)
```

Comment:
Right, there was this thing. Is it still experimental in JUnit 5?

From my understanding this only makes development inside Android Studio a bit more convenient, because you can run the class and all it's inner classes at once. But the usecase of the build system not finding the tests in not the case here with gradle

Reply:
I don’t know too much about it, I just found it to be a working way of grouping related test methods.

> this only makes development inside Android Studio a bit more convenient

This does seem to be the case; I just tried running tests from Gradle without it and they run just fine. I’ll remove the runner.